lib/cast.js

```diff
@@ -317,7 +314,6 @@ module.exports = function cast(schema, obj, options, context) {
                     context: context
                   });
                 }
-                continue;
```

Comment:
This seems to be in a deeply nested condition, not sure if removing it is okay, frankly, I prefer to just keep it for safety.

I'm curoius, how did you spot those unnecessary `continue` statements? Are you using some kind of tool?

Reply:
I took a deeper look into this, it should be fine to remove this `continue` as well.
Thanks 👍